src/ServiceControl.LoadTests.Reporter/Statistics.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ServiceControl.LoadTests.Reporter
+{
+    using System.Threading;
+
+    class Statistics
+    {
+        long audits;
```

Comment:
So the assumption here is when we have messages in the queue and we restart the processing the hostid will be different. we might still be consuming old audit messages but once we see the new hostid we assume we caught up and we will reset the stats. Correct?

Reply:
Yeah.